### PR TITLE
Copy edits

### PR DESCRIPTION
**Pull request by plink01001**

### CODE REVIEW

**Comment by nightwife** [comment]
@plink01001 I reviewed the edits per @sneak and nothing jumped out at me.  Edits are good.

File: whitepaper.md [hunk 1]
```diff
@@ -58,54 +58,54 @@ urlcolor: blue
...
 
-The fundamental unit of account on the Steem platform is STEEM, a crypto currency token. 
-Steem operates on the basis of one-STEEM, one-vote. Under this model, individuals who have contributed the most to the platform, as measured by their account balance, have the most influence over how contributions are scored. Furthermore, Steem only allows members to vote with STEEM when it is committed to a vesting schedule. Under this model, members have a financial incentive to vote in a way that maximises the long term value of their STEEM.  
+The fundamental unit of account on the Steem platform is STEEM, a cryptocurrency token. 
+Steem operates on the basis of one-STEEM, one-vote. Under this model, individuals who have contributed the most to the platform, as measured by their account balance, have the most influence over how contributions are scored. Furthermore, Steem only allows members to vote with STEEM when it is committed to a vesting schedule. Under this model, members have a financial incentive to vote in a way that maximizes the long term-value of their STEEM.
```
Comment (andrarchy) [inline whitepaper.md]: Long term-value? I can only find examples of either "long-term value" or "long term value"

Comment (plink01001) [reply whitepaper.md]: should be fixed now. nice catch!

**Review by andrarchy — APPROVED** [review]

**Review by jredbeard — APPROVED** [review]
lgtm

**Pull request merged**